### ISSUE TITLE
How do I load page with disabling images?

### ISSUE THREAD

**Issue by bondeanikets** [cobrateam/splinter#583]
I am doing this:

browser = Browser('firefox', profile_preferences=proxy_settings)
path = #Given the path
browser.visit(path)
soup = BeautifulSoup(browser.html, 'html.parser')

For faster performance, I want to load pages by disabling the images. I tried editing the "permissions.default.image" parameter to 2 in "about:config" of firefox, but it resets everytime firefox is under remote control.

TL;DR: How do I pass "permissions.default.image" in splinter.browser.visit() request?

Thanks

**Comment by fx86** [comment]
For doing this in Chrome, I opened chrome.py from the splinter directory under site-packages.

Then I added these lines after `options = ...` to make it appear like - 

```
        prefs = {"profile.managed_default_content_settings.images":2}
        options.add_experimental_option("prefs",prefs)
```
That seems to be working for me.

**Comment by impshum** [comment]
Full lists are here.  
http://kb.mozillazine.org/About:config_entries
http://kb.mozillazine.org/Category:Preferences

`with Browser(headless=False, incognito=True, profile_preferences={ 'permissions.default.image': 2}) as browser:`
